### PR TITLE
Added "enginecli setup-drop" command

### PR DESCRIPTION
**Pull request by zomglings**
## Changes

`enginecli setup-drop` makes it easier to create and set up claims on a `Dropper` contract.

## How to test these changes?

Tested manually running Polygon mainnet operations.

### CODE REVIEW

File: cli/enginecli/setup_drop.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import argparse
+
+from brownie import network
```
Comment (Andrei-Dolgolev) [inline cli/enginecli/setup_drop.py:3]: setup.py of api enviroment not depend on brownie anymore.
Maybe move to cli dir?

Comment (zomglings) [reply cli/enginecli/setup_drop.py:3]: @Andrei-Dolgolev : This is in `cli/`.

Comment (Andrei-Dolgolev) [reply cli/enginecli/setup_drop.py:3]: My miss i was think you cmmented out api because of it in api.

**Review by Andrei-Dolgolev — APPROVED** [review]
Looks correct

**Pull request merged**